app/src/main/res/values/strings.xml

```diff
@@ -25,6 +25,11 @@
   <string name="state_learn_again_button">LEARN AGAIN</string>
   <string name="topic_overview_see_more">See More</string>
   <string name="topic_download_text">(%s MB)</string>
+  <string name="overview">Overview</string>
```

Comment:
String names should have context or they won't be clear when referenced in code. Suggest: ``topic_tab_overview``, ``topic_tab_play``, etc.

Reply:
Removed